### PR TITLE
the names in xsd were all in low case and are fixed properly now

### PR DESCRIPTION
**Pull request by jianmingtu**
userToken is touched in xsd file

### CODE REVIEW

File: icon-two-common-models/src/main/resources/xsdSchemas/auth-models-1.xsd [hunk 1]
```diff
@@ -134,49 +134,49 @@
...
-            <xsd:element name="siteMinderTransactionID" nillable="true" type="xsd:string"/>
+            <xsd:element name="RemoteClientBrowserType" nillable="true" type="xsd:string"/>
+            <xsd:element name="RemoteClientHostName" nillable="true" type="xsd:string"/>
+            <xsd:element name="RemoteClientIPAddress" nillable="true" type="xsd:string"/>
+            <xsd:element name="UserIdentifier" nillable="true" type="xsd:string"/>
+            <xsd:element name="AuthoritativePartyIdentifier" nillable="true" type="xsd:string"/>
+            <xsd:element name="BiometricsSignature" nillable="true" type="xsd:string"/>
+            <xsd:element name="CSNumber" nillable="true" type="xsd:string"/>
+            <xsd:element name="SiteMinderSessionID" nillable="true" type="xsd:string"/>
+            <xsd:element name="SiteMinderTransactionID" nillable="true" type="xsd:string"/>
```
Comment (sylar-zeyangz) [inline icon-two-common-models/src/main/resources/xsdSchemas/auth-models-1.xsd:179]: These are string not object, please use camel-case

File: icon-two-common-models/src/main/resources/xsdSchemas/auth-models-1.xsd [hunk 2]
```diff
@@ -30,8 +30,8 @@
             <xsd:element name="latestPhoto" nillable="true" type="xsd:string"/>
```
Comment (sylar-zeyangz) [inline icon-two-common-models/src/main/resources/xsdSchemas/auth-models-1.xsd:30]: latestPhoto, firstName, lastName and csNum are not nillable

Comment (sylar-zeyangz) [reply icon-two-common-models/src/main/resources/xsdSchemas/auth-models-1.xsd:30]: you might want to be mindful about others fields when copy-pasting, some aren't nillable

**Review by sylar-zeyangz — CHANGES_REQUESTED** [review]

**Review by sylar-zeyangz — APPROVED** [review]

File: icon-two-common-models/src/main/resources/xsdSchemas/auth-models-1.xsd (hunk 2 above)
Comment (jianmingtu) [reply icon-two-common-models/src/main/resources/xsdSchemas/auth-models-1.xsd]: I checked. yes, Auth has some non-nillable properties in it.

**Pull request merged**